inference/core/env.py

```diff
@@ -224,7 +224,7 @@
 NUM_WORKERS = int(os.getenv("NUM_WORKERS", 1))
 
 ONNXRUNTIME_EXECUTION_PROVIDERS = os.getenv(
-    "ONNXRUNTIME_EXECUTION_PROVIDERS", "[CUDAExecutionProvider,CPUExecutionProvider]"
+    "ONNXRUNTIME_EXECUTION_PROVIDERS", "[CUDAExecutionProvider,OpenVINOExecutionProvider]"
```

Comment:
Should we leave `CPUExecutionProvider` as the last one incase something with the OpenVINO dependancies don't work?